Documentation/CMakeLists.txt

```diff
@@ -43,6 +43,17 @@ IF(DOXYGEN_FOUND)
     # Other parameters
     SET(DOC_OUTPUT_FILE "${COMPRESSED_HELP_TARGET_DIR}/${COMPRESSED_HELP_TARGET_FILE}")
     SET(DOC_GENERATION_CMAKE_FILE ${UNCOMPRESSED_HELP_TARGET_DIR}/GenerateDoc.cmake)
+
+    # If a Google Analytics tracking ID is set, configure and specify the modified footer file.
+    # otherwise, default Doxygen footer will be used
+    IF (PLUSAPP_DOCUMENTATION_GOOGLE_ANALYTICS_TRACKING_ID STREQUAL "")

```

Comment:
I would set ${FOOTER_FILE} to empty here (even if effectively it will be always empty without us explicitly setting it, just to make it more explicit).